site/content/docs/02-template-syntax.md

```diff
@@ -165,6 +165,19 @@ Comments beginning with `svelte-ignore` disable warnings for the next block of m
 <input bind:value={name} autofocus>
 ```
 
+---
+
+You can document components with the `@component` tag. Component comments will appear when hovering over a component in an editor using the [Svelte Language Server](https://github.com/sveltejs/language-tools).
+
+```sv
+<!--
+@component
+This is a component description.
+
+@example <HelloWorld />
```

Comment:
Perhaps `@example` should be mentioned in the text if it also has special meaning? Are there other tags we should mention as well?

Reply:
For simplicity, I've omitted the `@example` tag in the example.